robot/imu.c

```diff
@@ -17,24 +17,21 @@
 #define MAG_MODE 0x06               // magnetometer at 100Hz
 
 #define CALIBRATION_REPEAT 8 // how many times to repeat calibration to be sure
-#define MAGIC_REF 0xDEADBEAF
-#define TRUST_AZIMUTH 0.95
-
-// hard coded angle diff(X, north): needs to be updated in thevenin amphitheater
-#define X_NORTH_DIFF (-0.5)
+#define MAGIC_REF 0xFA7B00B5
+#define TRUST_AZIMUTH 0.97
 
 // RAM buffer to save calibration during page clear
 static float magBiasRAM[3], magScaleRAM[3];
 static unsigned int magicCodeRAM;
 
-
 // non volatile data
 static float magbias[3] __attribute__((section(".flashdata")));  // mag bias calibration data
 static float magscale[3] __attribute__((section(".flashdata"))); // mag scale calibration data
 static unsigned int magicCode __attribute__((section(".flashdata")));  // mismatch w/ MAGIC_REF -> need calibration
 
 static float magCalibration[3] = {0, 0, 0}; // Factory mag calibration
-
+// hard coded angle diff(X, north): needs to be updated in thevenin amphitheater
+static float AzimuthDiff = 0.0;
```

Comment:
Your style guide seems to mandate a lower case character at the beginning of a variable name.